contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,32 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks of GitHub repositories aren't updated automatically. In order to keep your fork of the repository up-to-date with the latest changes and avoid merge conflicts, you should update it regularly.
```

Comment:
In order to -> To

I feel "fork of ... repository" being so close to each other is a bit redundant. It can be inferred that the fork is of the repository as in the previous sentence, we say "Forks of GitHub repositories".

```suggestion
Forks of GitHub repositories aren't updated automatically. To keep your fork up-to-date with the latest changes and avoid merge conflicts, you should update it regularly.
```

This also reduces a 28 syllable clause to 19.